pulsar-client/src/main/java/org/apache/pulsar/client/impl/PulsarClientImpl.java

```diff
@@ -791,7 +791,7 @@ private static EventLoopGroup getEventLoopGroup(ClientConfigurationData conf) {
         return EventLoopUtil.newEventLoopGroup(conf.getNumIoThreads(), threadFactory);
     }
 
-    private static ThreadFactory getThreadFactory(String poolName) {
+    public static ThreadFactory getThreadFactory(String poolName) {
```

Comment:
Actually, it makes more sense to just move this static method over to the other class.

Reply:
changed not needed anymore